src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -48,10 +50,11 @@ public class EditCommand extends Command {
             + "[" + PREFIX_NAME + "NAME] "
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
-            + "[" + PREFIX_ROOM_NUMBER + "ROOMNUMBER] "
+            + "[" + PREFIX_ROOM_NUMBER + "ROOM NUMBER] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
             + "[" + PREFIX_EMERGENCY_NAME + "EMERGENCY_NAME] "
             + "[" + PREFIX_EMERGENCY_PHONE + "EMERGENCY_PHONE] "
+            + "[" + PREFIX_GRAD_YEAR + "GRADUATION YEAR] "
```

Comment:
Would it be better to name these ROOM_NUMBER and GRADUATION_YEAR to standardize the naming?